release/mixer.sh

```diff
@@ -161,17 +165,41 @@ generate_mix() {
 
     build_update "${mix_ver}"
 
+    if [[ ${SIGN_MOM} == "true" ]]; then
+        sign_mom
+    fi
+
     build_deltas "${mix_ver}"
 
     echo -n "${mix_ver}" | sudo -E tee update/latest > /dev/null
 }
 
+sign_mom(){
+
+    if [ -z "${CUSTOM_MoM_CMD}" ]; then
+        warn "Skipping stage."
+        warn "No custom sign command for MoM is provided"
```

Comment:
if you use SIGN_MOM and CUSTOM_MoM_SIGN, than this cannot be a warn, it must be an error.

But I think you can do everything with a single var, see other comments.